package.json

```diff
@@ -20,10 +20,11 @@
   "dependencies": {
     "aes-js": "^3.1.2",
     "bfx-api-node-rest": "^4.1.2",
-    "bfx-hf-algo": "^4.3.0",
+    "bfx-hf-algo": "github:tarcisiozf/bfx-hf-algo#change-gid-generation",
```

Comment:
not sure but is some change from change-gid-generator branch used here?

Reply:
hosted uses hf-algo via the hf-server, this way we can test hosted with the most recent code.